Contributing.md

```diff
@@ -0,0 +1,15 @@
+### One issue or bug per Pull Request
```

Comment:
This file was stolen from [Relativity](https://github.com/dfed/Relativity/blob/master/Contributing.md). I'm aware I broke my own rule on small PRs 😬